Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -3437,6 +3441,24 @@ PartialResult WARN_UNUSED_RETURN BBQJIT::addTry(BlockSignature signature, Stack&
     return { };
 }
 
+PartialResult WARN_UNUSED_RETURN BBQJIT::addTryTable(BlockSignature signature, Stack& enclosingStack, const Vector<CatchHandler>& targets, ControlType& result, Stack& newStack)
+{
+    m_usesExceptions = true;
+    ++m_tryCatchDepth;
+    ++m_callSiteIndex;
+    result = ControlData(*this, BlockType::TryTable, signature, currentControlData().enclosedHeight() + currentControlData().implicitSlots() + enclosingStack.size() - signature->argumentCount());
+    result.setTryInfo(m_callSiteIndex, m_callSiteIndex, m_tryCatchDepth);
+    currentControlData().flushAndSingleExit(*this, result, enclosingStack, true, false);
+
+    m_pendingTryTables.append(targets);
```

Comment:
Instead of using `m_pendingTryeTables` member variables to track tables, how about storing it to `ControlData`?